Source/WebKit/Platform/spi/ios/UIKitSPI.h

```diff
@@ -1262,12 +1262,13 @@ typedef struct {
 @property (nonatomic, assign, readwrite) BOOL shouldEvaluateForInputSystemHandling;
 @end
 
-#endif // HAVE(UI_ASYNC_TEXT_INTERACTION)
-
-@interface UIResponder (Staging_118307558)
-- (void)replace:(id)sender;
+@protocol UIAsyncTextInputDelegate_Staging<UIAsyncTextInputDelegate>
+- (void)invalidateTextEntryContext; // Added in rdar://118536368.
+- (void)replaceText:(id)sender; // Added in rdar://118307558.
```

Comment:
Do we have any tracking radars for removing any of this staging?

Reply:
No — I'm planning to do a pass over all these staging defines, after the names are all finalized in the SDK.